addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -87,8 +88,8 @@ export class MassMailingHtmlField extends HtmlField {
         popover.style.left = leftPosition + 'px';
     }
 
-    async commitChanges() {
-        if (this.props.readonly || !this.isRendered) {
+    async commitChanges({ shouldInline = true } = {}) {
+        if (this.props.readonly || !this.isRendered || !shouldInline) {
```

Comment:
I'm confused by this `shouldInline` parameter. When we say "you should not inline" then it calls `commitChanges` no matter what but when we say "you should inline" then it tests other conditions to call `commitChanges`. I'm completely lost here.